index.bs

```diff
@@ -2015,8 +2015,9 @@ Methods</h4>
 				rendering <code>length</code> sample-frames of audio into
 				{{[[rendered buffer]]}}
 
-				<li>For every <a>render quantum</a>, check and suspend
-				rendering if necessary.
+				<li>For every <a>render quantum</a>, check and
+				<a href="#dom-offlineaudiocontext-suspend">
```

Comment:
Maybe use bikeshed-style link?  I think it's `{{OfflineAudioContext/suspend()}}`